crm_opportunity_currency/models/crm_lead.py

```diff
@@ -0,0 +1,66 @@
+# Copyright 2018 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+from odoo import api, fields, models
+
+
+class CrmLead(models.Model):
+    _inherit = 'crm.lead'
+
+    customer_currency_id = fields.Many2one(
+        string='Customer currency',
```

Comment:
In v12 when the string is the same that the field name, this parameter is not necessary.

And if it has `_id` or `_ids`, that is not added in the string.

Could you remove this parameter, please?

Reply:
But it doesn't hurt to have it explicitly stated ?

Reply:
it does not hurt at all